pkg/constants/constants.go

```diff
@@ -16,4 +16,9 @@ const (
 	UpstreamHostLabel = "upstream-host"
 	// RegistryCachePort is the port on which the pull through cache serves requests.
 	RegistryCachePort = 5000
+
+	// RemoteURLAnnotation is an annotation on registry cache Service which denotes the upstream registry URL
+	RemoteURLAnnotation = "remote-url"
+	// RemoteHostAnnotation is an annotation on registry cache Service which denotes the upstream registry host and optionally a port.
+	RemoteHostAnnotation = "remote-host"
```

Comment:
Shouldn't the annotation key be `upstream` as it is at the end used to be set as upstream field value in the RegistryCacheStatus?